backend/typescript/middlewares/validators/camperValidators.ts

```diff
@@ -13,24 +13,43 @@ export const createCampersDtoValidator = async (
   res: Response,
   next: NextFunction,
 ) => {
-  if (!Array.isArray(req.body) || req.body.length === 0) {
+  const { campers } = req.body;
+  const { campSessions } = req.body;
+  const waitlistId = req.query?.wId;
+
+  if (!Array.isArray(campers) || campers.length === 0) {
     return res
       .status(400)
       .send(
-        "No campers sent - there must be at least one camper in the request array.",
+        "No campers sent - there must be at least one camper in the campers array field.",
       );
   }
-  const { campSession, chargeId } = req.body[0];
+  if (!Array.isArray(campSessions) || campSessions.length <= 0) {
```

Comment:
I think less than `<=0` is correct. We want to check that at least 1 session is provided so if the length is 0, we want to return an error saying that we need at least 1 session data in the array. Alternatively I can just do `=== 0` since length can never be less than 0 tbf. Having it as `<=1` would throw the error even when 1 session is passed in

Reply:
Ahh I was getting confused because the same thing was checked via `===` in the previous conditional